_includes/specials/taproot/en/10-ln-with-taproot.md

```diff
@@ -0,0 +1,198 @@
+importantly **has the correct address**.  Checking the address helps
+ensure that it is difficult to spam the channel gossip mechanism; you
+need actual funds on the blockchain in order to send channel gossip.
+Thus, in practice, even P2TR channels require some amount
+of remote compatibility; otherwise, senders will ignore these channels
+for routing, as they cannot validate that those channels *exist*.
+
+### Time Frames
+
+I think the best way to create time frames for features on a distributed
+FOSS project is to look at *previous* features and how long they took,
+and use those as the basis for how long features will take to actually
+deploy.[^planning-details]
+
+The most recent new major feature that I believe is similar in scope to
+PTLCs over LN is [dual-funding][topic dual funding].  Lisa Neigut created an initial
+proposal for a dual-funding protocol in [BOLTs #524][], with the [first
+dual-funded channel on mainnet][neigut first dual funded] being
+[opened][first dual funded tx] almost 2 years and 6 months later.
+Dual-funding only requires compatibility with your direct peers.  PTLCs
+over LN require compatibilty with all routing nodes on your
+selected paths, including the receiver, so I feel justified in giving
+this feature a +50% time modifier due to the added complication, for an
+estimate of 3 years and 9 months starting from when a specific PTLC
+protocol is proposed.
+
+For P2TR channels, we should note that while this is "only"
+between two direct peers, it also has lower benefits.  Thus, I expect it
+will be lower priority.  Assuming most developers prioritize
+PTLC-over-LN, then I expect P2TR channels will start
+getting worked on by the time the underlying [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] or other
+ways to implement Decker-Russell-Osuntokun ("[Eltoo][topic eltoo]") are available.
+
+[^route-randomization]:
+    A payer can choose a very twisty path (i.e. route randomization) to
+    make HTLC correlation analysis wrong, but that has its own drawbacks:
+
+    * Twisty paths are costlier *and* less reliable (more nodes
+      have to be paid, and more nodes need to *successfully* forward
+      in order for the payment to reach the destination).
+    * Twisty paths are longer, meaning the payer is telling *more*
+      nodes about the payment, making it *more* likely they will hit
+      *some* surveilant node.
+      Thus, twisty paths are not necessarily a perfect improvement
+      in privacy.
+
+[^planning-details]:
+    Yes, details matter, but they also do not: from a high enough
+    vantage point, the unexpected hardships of some aspect of
+    development and the unexpected non-hardships of other aspects
+    of development cancel out, and we are left with every major
+    feature being roughly around some average time frame, with
+    any unexpectedly high bonus or malus being "just noise".
+
+    Yes, there are more developers now, that also means we need
```

Comment:
I'm not sure how important this paragraph is to the main focus of the article. I'd be tempted to drop it.